dev_tools/qualtran_dev_tools/bloq_finder.py

```diff
@@ -121,10 +121,16 @@ def get_bloq_examples() -> List[BloqExample]:
     bloqs_root = reporoot / 'qualtran/bloqs'
     paths = get_bloq_module_paths(bloqs_root)
 
+    # These examples are too large to run.
+    REJECTED_BEXAMPLES = set(
+        ["qubitization_qpe_hubbard_model_large", "qubitization_qpe_hubbard_model_small"]
+    )
+
```

Comment:
Can we just remove them from the serialization test instead of removing completely?

Reply:
+1; `get_bloq_examples` should get all the bloq examples. If there are specific places where they shouldn't be used, you can intercept there.

Reply:
Fixed